_examples/how-to-create-a-flutter-app-with-serverless.md

```diff
@@ -0,0 +1,402 @@
+
+{%change%} Run the following in the project root.
+
+```bash
+$ flutter create frontend
+$ cd frontend
+```
+
+This sets up our flutter app in the `frontend/` directory.
+
+We also need to load the environment variables from our SST app. To do this, we'll be using the [`flutter_dotenv`](https://pub.dev/packages/flutter_dotenv) package.
+
+{%change%} Install the `flutter_dotenv` package by running the following in the `frontend/` directory.
+
+```bash
+$ flutter pub add flutter_dotenv
+```
+
+Create a `.env` file in root and create a variable to hold the API endpoint
+
+```
+API_URL=https://sez1p3dsia.execute-api.us-east-1.amazonaws.com
+```
+
+Add the `.env` file to your assets bundle in `pubspec.yaml`. Ensure that the path corresponds to the location of the .env file!
+
+```yaml
+assets:
+  - .env
+```
+
+Let's start our flutter development environment.
+
+{%change%} In the `frontend/` directory run.
+
+```bash
+$ flutter run
+```
+
+This will open up an emulator and the app will be loaded
+
+### Add the click button
+
+We are now ready to add the UI for our app and connect it to our serverless API.
+
+{%change%} Replace `frontend/lib/main.dart` with.
+
+```dart
+import 'package:flutter/material.dart';
+import 'package:flutter_dotenv/flutter_dotenv.dart';
+import 'package:http/http.dart' as http;
+
+Future main() async {
+  await dotenv.load(fileName: ".env");
+  runApp(MyApp());
+}
+
+class MyApp extends StatefulWidget {
+  MyApp({Key? key}) : super(key: key);
+
+  @override
+  State<MyApp> createState() => _MyAppState();
+}
+
+class _MyAppState extends State<MyApp> {
+  update() async {
+    Uri uri = Uri.parse(dotenv.env['API_URL']!);
+    var result = await http.post(uri);
+    print(result.body);
+    setState(() {
+      counter = int.parse(result.body);
+    });
+    print(counter);
+  }
+
+  int counter = 0;
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Flutter Demo',
+      theme: ThemeData(
+        primarySwatch: Colors.blue,
+      ),
+      home: Scaffold(
+        appBar: AppBar(
+          title: Text("Flutter serverless-stack demo"),
+        ),
+        body: Container(
+          child: Center(
+            child: Column(
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: [
+                Text("This button is pressed $counter times"),
+                MaterialButton(
+                  onPressed: () {
+                    setState(() {
+                      update();
+                    });
+                  },
+                  child: Text(
+                    "Click Me",
+                    style: TextStyle(color: Colors.white),
+                  ),
+                  color: Colors.blue.shade500,
+                ),
+              ],
+            ),
+          ),
+        ),
+      ),
+    );
+  }
+}
+```
```

Comment:
Got an error the `http` package is not found

<img width="728" alt="Screen Shot 2021-11-04 at 3 59 36 AM" src="https://user-images.githubusercontent.com/83515/140277888-9b03b873-3ae6-4986-a06e-ebbbe349f062.png">